src/plugins/vis_type_timeseries/server/plugin.ts

```diff
@@ -48,8 +50,9 @@ export interface VisTypeTimeseriesSetup {
   getVisData: (
     requestContext: VisTypeTimeseriesRequestHandlerContext,
     fakeRequest: FakeRequest,
-    options: GetVisDataOptions
-  ) => ReturnType<GetVisData>;
+    // ideally this should be VisPayload type, but currently has inconsistencies with x-pack/plugins/infra/server/lib/adapters/framework/kibana_framework_adapter.ts
```

Comment:
looks like @DziyanaDzeraviankina already fixed that issue here https://github.com/elastic/kibana/pull/91977

Reply:
I would not like to make any changes in infra code in this PR, let's ask @DziyanaDzeraviankina to remove this `any` and comment when resolving merge conflicts?